src/routers/genericRouters/routerFilterBy.ts

```diff
@@ -55,7 +55,7 @@ export class RouterFilterBy implements Router {
       }
 
       if (queryParams[element] !== undefined) {
-        if (this.filterByRange.includes(element)) {
+        if (this.filterByRange.includes(element) || element.startsWith('annotations.freq')) {
```

Comment:
Should it be && not ||?

Reply:
this is an ||, we can't have dot notation in the schema file yet. This is a special case that our schema doesn't support.